promql/promqltest/testdata/functions.test

```diff
@@ -892,90 +892,90 @@ eval instant at 55s avg_over_time(metric[1m])
 eval instant at 55s sum_over_time(metric[1m])/count_over_time(metric[1m])
   {} 3
 
-eval instant at 1m avg_over_time(metric2[1m])
+eval instant at 55s avg_over_time(metric2[1m])
```

Comment:
Sorry I'm having trouble imagining this.
At 55s, `[1m]` starts in negative time?

Reply:
@bboreham is the above explanation satisfactory?

Reply:
(to enforce this, have an example, we can add some promqltests with negative timestamps as well, if we don't already have any, they could act as good "fuzz" tests as well)

Reply:
I think the `load` comment always starts at timestamp 0. Thinking about it, it is quite hard to create test series at arbitrary timestamps. You could skip the sample at t=0 with a `_` and then set an interval for the arbitrary timestamp you want. It probably wouldn't accept a negative interval… But that's a story for another day…